src/Chapter1.hs

```diff
@@ -631,8 +648,15 @@ Implement a function that returns the sum of the last two digits of a number.
 Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
-
-sumLast2 n = error "sumLast2: Not implemented!"
+sumLast2 :: Int -> (Int)
+sumLast2 n  
+  | n < 1000 =
+    let lastN = (n `mod` 10)
+        previousN 
+          | n < 100 = (n `div` 10)
+          | n < 1000 = ((n `mod` 100) `div` 10)
+    in (lastN + previousN)
+  | otherwise = error "Number is out of range. Choose a number smaller than 1000"
```

Comment:
Is there any specific reason to throw an error on numbers bigger than 1️⃣ 0️⃣ 0️⃣ 0️⃣ ?
I believe, the function should work for them as well 👍🏻